src/Microsoft.DocAsCode.Build.RestApi/RestApiDocumentProcessor.cs

```diff
@@ -129,11 +129,35 @@ public override SaveResult Save(FileModel model)
                 LinkToUids = model.LinkToUids,
                 FileLinkSources = model.FileLinkSources,
                 UidLinkSources = model.UidLinkSources,
+                XRefSpecs = GetXRefInfo(vm, model.Key).ToImmutableArray()
             };
         }
 
         #region Private methods
 
+        private static IEnumerable<XRefSpec> GetXRefInfo(RestApiRootItemViewModel rootItem, string key)
+        {
+            yield return new XRefSpec
+            {
+                Uid = rootItem.Uid,
+                Name = rootItem.Name,
+                Href = key,
+            };
+
+            if (rootItem.Children != null)
+            {
+                foreach (var child in rootItem.Children)
+                {
+                    yield return new XRefSpec
+                    {
+                        Uid = child.Uid,
+                        Name = child.OperationId,
+                        Href = key,
+                    };
+                }
+            }
```

Comment:
yes should add tag

---
In reply to: [86943253](https://github.com/dotnet/docfx/pull/870#discussion_r86943253) [](ancestors = 86943253)